### ISSUE TITLE
Audio playing/streaming problem with AppImage build

### ISSUE THREAD

**Issue by ghost** [skyjake/lagrange#623]
So, with the launch of my music and public radio service, I have been given feedback from users on various problems with Lagrange on platforms other than Windows: Lagrange doesn't seem to play and/or stream the audio on Android or iOS or the AppImage on Linux.

Here is a link you can use to test: gemini://hashnix.club/cgi/radio.cgi
There is also this one: gemini://auragem.letz.dev/music/stream/public_radio (always streams audio/mpeg)

**Comment by skyjake** [comment]
It sounds like a build config issue if the AppImage doesn't play or stream audio on Linux. I will need to check that.

When it comes to the mobile betas, audio playback remains a work in progress and problems with it are covered by the already existing tracker issues (here/BBS).

**Comment by skyjake** [comment]
See also this comment:

gemini://bbs.geminispace.org/u/skyjake/5424

**Comment by skyjake** [comment]
I can reproduce this issue when running the AppImage on XUbuntu. The specific error message is:
```
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave
```
It is not immediately clear to me how to change the build to fix this.

**Comment by skyjake** [comment]
Fixed for v1.17. Around version 1.16.4 I started using a custom-built SDL in the AppImage, but its build settings were such that the audio dependencies were not being included with the executable. Hence no audio. I fixed it by changing the settings (`SDL_LOADSO`) and made sure all the audio drivers are present.
